src/wallet/api/pending_transaction.cpp

```diff
@@ -78,7 +79,6 @@ std::vector<std::string> PendingTransactionImpl::txid() const
 
 bool PendingTransactionImpl::commit(const std::string &filename, bool overwrite)
 {
-
```

Comment:
Spurious

Reply:
thought it unnecessary :) bring it back?

Reply:
This kind of cosmetic changes are generally discouraged. Minimizing the number of changed lines is much more important.

Reply:
ok, thank you. i'll take this into account in future